sailthru_content/services/tests/test_sailthru_translation_service.py

```diff
@@ -70,7 +71,7 @@ def test_translate_programs(self):
         self.assertEqual(len(translated_programs[0]['vars']['course_runs']), 4)
 
     def _get_expected_sailthru_item(self, seat_type=None):
```

Comment:
It seems we have tests; but, they aren't actually being run by Travis. That should be fixed.